specification/dartLangSpec.tex

```diff
@@ -16880,13 +16880,13 @@ \subsection{If}
 
 \rationale{%
 Under reasonable scope rules such code is problematic.
-If we assume that \code{v} is declared 
```

Comment:
Seems my "remove trailing spaces on save" has caught some dangling spaces.

Reply:
Trailing spaces are bad, m'kay?

Reply:
Certainly!

I'm saying that there are no trailing spaces in `dartLangSpec.tex` today. In general, there shouldn't be many commits where there are any trailing spaces, because I usually remember to remove them, but I can see that this PR starts from a commit where there are quite a few trailing spaces.

But this means that removing trailing spaces in this PR will not remove any trailing spaces from the result of landing this PR (they're gone already), it will just make the merging operation a bit more complex.

I was worried that this kind of "fix something that has been fixed" change could in some situations give rise to a reversal: A line here and there could suddenly be edited back to an earlier version, because it has been changed in this PR, and it contains the old text in this PR (minus that trailing space). So `git` thinks that the old text is what we really want, and replaces the current text by the old text from this PR.

That's the reason why I suggested doing a `rebase` on this PR.